code/src/components/app-typography/AppTypography.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { render, screen } from "@testing-library/react";
+import AppTypography, {
+  AppTypographyVariant
+} from "@/components/app-typography/AppTypography";
+import { FormattedMessage } from "react-intl";
```

Comment:
don't forget about our import convention
- libs
- components,containers...
- styles

should to move above and follow convention

Reply:
Thanks, fixed